packages/wdio-cli/src/interface.js

```diff
@@ -208,21 +192,47 @@ export default class WDIOCLInterface extends EventEmitter {
         )
 
         this.updateClock()
+    }
 
-        if (this.sigintTriggered) {
-            clearTimeout(this.interval)
-            this.interface.log('\n')
-            this.interface.log(!isFinished
-                ? 'Ending WebDriver sessions gracefully ...\n' +
-                  '(press ctrl+c again to hard kill the runner)'
-                : 'Ended WebDriver sessions gracefully after a SIGINT signal was received!'
-            )
+    printReporters() {
+        const totalJobs = this.totalWorkerCnt
+
+        this.interface.clearAll()
+
+        /**
+         * print reporter output
+         */
+        for (const [reporterName, messages] of Object.entries(this.messages.reporter)) {
+            this.interface.log(chalk.bgYellow.black(`"${reporterName}" Reporter:`))
+            this.interface.log(messages.join(''))
+            this.interface.log()
         }
 
-        if (isFinished) {
-            clearTimeout(this.interval)
-            this.interface.log('\n')
+        /**
+         * print stdout and stderr from runners
+         */
+        /* istanbul ignore else */
+        if (this.interface.stdoutBuffer.length) {
+            this.interface.log(chalk.bgYellow.black('Stdout:\n') + this.interface.stdoutBuffer.join(''))
+        }
+        /* istanbul ignore else */
+        if (this.interface.stderrBuffer.length) {
+            this.interface.log(chalk.bgRed.black('Stderr:\n') + this.interface.stderrBuffer.join(''))
+        }
+        /* istanbul ignore else */
+        if (this.messages.worker.error) {
+            this.interface.log(chalk.bgRed.black('Worker Error:\n') + this.messages.worker.error.map(
+                (e) => e.stack
+            ).join('\n') + '\n')
         }
+
+        this.interface.log(
```

Comment:
Seems like this does about the same thing as the code on line 145:

`
            return this.interface.log(
                `${clockSpinnerSymbol} ` +
                `${this.jobs.size} running, ` +
                `${this.result.passed} passed, ` +
                `${this.result.failed} failed, ` +
                `${totalJobs} total ` +
                `(${Math.round((this.result.finished / totalJobs) * 100)}% completed)`)
`

Reply:
Fixed.